lib/async_request.rb

```diff
@@ -1,4 +1,42 @@
-require "async_request/engine"
+require 'async_request/engine'
 
 module AsyncRequest
+  VALID_ALGORITHMS = %w[HS256 RS256].freeze
+
+  @config = {
+    sign_algorithm: 'HS256',
+    encode_key: 'CHANGE_ME_IN_THE_INITIALIZER_FILE',
```

Comment:
I would change this to nil and explode if the user doesn't set them. If not you may be functional with security issues